src/test/resources/org/powerflows/dmn/io/yaml/reference_multiple.yml

```diff
@@ -0,0 +1,83 @@
+id: some_table_id_1
```

Comment:
Can we use hamburger-case of naming convention for resources?

Reply:
you mean kebab case :)
I prefer snake case for resources like constants or long numbers in Java.
It's more readable since word separation is standing out, unlike dash (-) separation which make me squint to parse the name...

Reply:
Ahh, yes. Kebab :-) Imho kebab looks better, for both directories and files.

Reply:
not for me...

Reply:
but lets have it...